src/components/atoms/ChatMessage/ChatMessage.scss

```diff
@@ -45,6 +45,43 @@ $reply-icon-radius: 20px;
 
   &__text {
     font-size: $font-size--md;
+
+    img {
+      max-width: 100%;
+    }
+
+    p {
+      margin-bottom: 0;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4,
+    h5,
+    h6 {
+      margin-bottom: 0.1rem;
+      line-height: 1.05;
+    }
+
+    h1 {
+      font-size: 1.5em;
+    }
+    h2 {
+      font-size: 1.4em;
+    }
+    h3 {
+      font-size: 1.3em;
+    }
+    h4 {
+      font-size: 1.2em;
+    }
+    h5 {
+      font-size: 1.1em;
+    }
+    h6 {
+      font-size: 1em;
+    }
```

Comment:
I think it might be better to just turn off the ability to set headings in the chat altogether using the `allowHeadings=false` option added in this review's suggestions.

```suggestion
```